pkg/kubelet/dockershim/helpers_windows.go

```diff
@@ -26,27 +26,16 @@ import (
 	dockercontainer "github.com/docker/docker/api/types/container"
 	dockerfilters "github.com/docker/docker/api/types/filters"
 	"github.com/golang/glog"
-	"k8s.io/api/core/v1"
 	runtimeapi "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
 )
 
 func DefaultMemorySwap() int64 {
 	return 0
 }
 
-func (ds *dockerService) getSecurityOpts(containerName string, sandboxConfig *runtimeapi.PodSandboxConfig, separator rune) ([]string, error) {
-	hasSeccompSetting := false
-	annotations := sandboxConfig.GetAnnotations()
-	if _, ok := annotations[v1.SeccompContainerAnnotationKeyPrefix+containerName]; !ok {
-		_, hasSeccompSetting = annotations[v1.SeccompPodAnnotationKey]
-	} else {
-		hasSeccompSetting = true
-	}
-
-	if hasSeccompSetting {
-		glog.Warningf("seccomp annotations found, but it is not supported on windows")
-	}
-
+func (ds *dockerService) getSecurityOpts(seccompProfile string, separator rune) ([]string, error) {
+	// return nil so docker will load the default seccomp profile
+	glog.Warningf("seccomp annotations are not supported on windows")
```

Comment:
@yujuhong : did you see this? https://github.com/kubernetes/kubernetes/blob/b32639f9e7d47d0f69bb41b5535803ae4c9d835a/pkg/kubelet/dockershim/helpers_unsupported.go#L33

With the recent changes, the first parameter to that method is the `seccompProfile`, not `containerName` Also, please look at how the calls to getSecurityOpts are made from docker_container.go and docker_sandbox.go

Thanks,
Dims